### ISSUE TITLE
Can it be counted only when the call is successful?

### ISSUE THREAD

**Issue by jianmaikj** [dream11/kong-scalable-rate-limiter#10]
### Summary

SUMMARY_GOES_HERE

**Comment by chirag-manwani** [comment]
Yes, this is a valid point. We could check for response status and increment the counter only if the status is 2XX. Is there any other case where the response is non 2XX but we still want to increment the request count? 
We could add a setting in the plugin schema as well which could be used to turn this behaviour on or off. 

@jianmaikj
